### PR TITLE
Remove debug log line from cgroup-parent feature e2e test.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: integration-cli/docker_cli_run_unix_test.go [hunk 1]
```diff
@@ -114,7 +114,6 @@ func getCgroupPaths(test string) map[string]string {
 	for _, line := range strings.Split(test, "\n") {
 		parts := strings.Split(line, ":")
 		if len(parts) != 3 {
-			fmt.Printf("unexpected file format for /proc/self/cgroup - %q\n", line)
```
Comment (duglin) [inline integration-cli/docker_cli_run_unix_test.go]: Can you move this func() to utils.go ?

**Comment by jfrazelle** [comment]
@GordonTheTurtle you are the worst this is vishh you knowww him

**Comment by jfrazelle** [comment]
LGTM

File: integration-cli/docker_cli_run_unix_test.go (hunk 1 above)
Comment (vishh) [inline integration-cli/docker_cli_run_unix_test.go]: Good idea. Done.

**Comment by duglin** [comment]
@vishh gofmt

**Comment by vishh** [comment]
@duglin: Oops. Fixed.

**Comment by duglin** [comment]
LGTM

**Pull request merged**